fuzz/Cargo.toml

```diff
@@ -9,16 +9,19 @@ edition = "2018"
 cargo-fuzz = true
 
 [dependencies]
-libfuzzer-sys = "0.4.0"
-bytes = "1"
-ckb-vm = { path = "..", features = ["asm"] }
+libfuzzer-sys = "0.4"
+spike-sys = "0.1.0"
+
+[dependencies.ckb-vm]
+path = ".."
+features = ["asm"]
 
 # Prevent this from interfering with workspaces
 [workspace]
 members = ["."]
 
 [[bin]]
-name = "asm"
```

Comment:
Why do we replace one with the other, instead of keeping both fuzzing targets? I thought they can work for different use cases.